src/PockyBot.NET.Persistence/Repositories/PegRepository.cs

```diff
@@ -15,7 +15,7 @@ public PegRepository(DatabaseContext context)
         public async Task<Peg> CreatePeg(Peg peg)
         {
             var added = _context.Add(peg);
-            await  _context.SaveChangesAsync();
+            await  _context.SaveChangesAsync().ConfigureAwait(false);
```

Comment:
Gotta be honest, I've just spent like 10 minutes reading up on it trying to figure out what this actually does, and I'm still not really clear...